Makefile

```diff
@@ -58,8 +61,8 @@ format-check:
 
 setup: setup_env build
 
-superuser: 
+superuser:
 	docker-compose exec web ./manage.py createsuperuser
 
-tail-logs: 
-	docker-compose logs -f web
\ No newline at end of file
+tail-logs:
+	docker-compose logs -f web
```

Comment:
my editor deletes spaces at the end of lines. also makes sure each file ends in a new line.

Reply:
Glad to see them go!